### ISSUE TITLE
Nextcloud client should not repetitively open account configuration window when launched in background (GNOME + Dbus service)

### ISSUE THREAD

**Issue by zeten30** [nextcloud/desktop#3786]
### Expected behaviour
- Client launched autoatically vi dbus service - > process /usr/bin/nextcloud --background
- configuration or other UI windows opens only on user request
- there should be an 'Do not show this again' checkbox

### Actual behaviour
- repetitively opens account configuration window
- no easy way to stop client when not needed

### Steps to reproduce
1. run client in background
2. use desktop
3. account configuration window appears from time time

### Client configuration
Client version:
3.3.3

Operating system:
Fedora 34/35 - Gnome

OS language:
EN_US

Client package (From Nextcloud or distro) (Linux only):
nextcloud-client-libs-3.3.3-1.fc34.x86_64
nextcloud-client-3.3.3-1.fc34.x86_64
nextcloud-client-nautilus-3.3.3-1.fc34.x86_64

![nextcloud-should-not-open-windows-by-default](https://user-images.githubusercontent.com/1825873/133070247-08ce3f2b-4bcb-4a96-a31f-d8bfeebd9c3f.png)

**Comment by FlexW** [comment]
Is your complaint that the "Add Nextcloud account" window pos up from time to time? That window should only pop up if you have no account configured.

Because Fedora uses GNOME and GNOME does not want to implement tray icons it is a little bit difficult to exit the Nextcloud client or disable autostart if you do not have an account configured.

I'm not sure we will win much with using dbus activation. I think the current autostart feature is fine enough.

**Comment by zeten30** [comment]
Yes, I want to avoid window popup if there's no account. It's fine to appear once, but I think that a checkbox 'do not show this again' will be very handy in this case. 

You can disable / mask dbus service unit, but it's not user friendly.

**Comment by FlexW** [comment]
I think dbus activation and the tick box on this dialog are two different issues. I think this ticket is for the tick box in the account wizard. You can open a new ticket for the dbus activation or better directly provide a pull request for it :) I think we would happily accept that feature, but it has at the moment no priority for us.

**Comment by bbkz** [comment]
Hi there,

i hope i am not off topic. I have ~300 Fedora Clients with some users that may use or could potentially use nextcloud in future.

I provided the nextcloud-client as a stock package, but since the dbus implementation all users that don't use nextcloud have the mentioned popup each time they login. The only workaround we came up with was

```
mkdir -p ~/.local/share/dbus-1/services
grep -v ^Exec /usr/share/dbus-1/services/com.nextcloudgmbh.Nextcloud.service > ~/.local/share/dbus-1/services/com.nextcloudgmbh.Nextcloud.service
```

for the users not using nextcloud, if ever they would like to use it in future `~/.local/share/dbus-1/services/com.nextcloudgmbh.Nextcloud.service` needs to be removed.

So the users needs to ask the IT support and then receive the fix ether to be run from command line or the support does it trough ssh.

This is not optimal, maybe there is a better solution for this problem?

A 'Do not show this again' checkbox whould already be a massive improvment.

Yours,
bbk

**Comment by mstenta** [comment]
Bump. There needs to be a way to turn this off. Not every user of a box is going to be using Nextcloud. But installing `nextcloud-desktop` causes this popup to show on every login, with no way to hide it.
